docs/aaaCtl/Billing.md

```diff
@@ -0,0 +1,23 @@
+# Billing
+
+Billing information can only be added by Organization Admins or by Users who have been granted read/write access to the billing option by Organization Admins. To grant read/write permissions for a User, go to the Users section of aaaCtl. Click on the Manage button next to the User's name. Under Permissions, check the write box for Billing.
+   ![](img/billingwrite.png)
+
+To enter billing information, click on Billing below your Organization name.
+   ![](img/billing.png)
+
+Click the Add button to add a new billing contact.
+   ![](img/addbilling.png)
+
+Enter your name, email, address and credit card information. Read the [Terms of Service](https://www.fullctl.com/legal#section=collapseToS) and check the box. Click Continue.
+Note: If a second billing contact and credit card is added for your Organization, the system will automatically charge the newest credit card added. If you would like a different card to be charged please contact support@fullctl.com.
```

Comment:
support@fullctl.com needs to be linked.
To do this please change the line to be `"... please contact <support@fullctl.com>."`

Reply:
@sunbuch you can re-request a review. 
![image](https://github.com/fullctl/docs.fullctl.com/assets/98209208/c3a4aaf4-eef5-46fe-973f-41ba3799e28e)
and also resolve this conversation 
![image](https://github.com/fullctl/docs.fullctl.com/assets/98209208/47ab1239-162c-421c-b4af-ab3c69c6bf19)